setup.cfg

```diff
@@ -38,7 +38,6 @@ setup_requires =
 APNS =
 	apns2>=0.3.0
 	importlib-metadata;python_version < "3.8"
-	pywebpush>=1.3.0
 	Django>=2.2
```

Comment:
Wont this throw an import error? You are still importing pywebpush in webpush.py

Reply:
It's my understanding on this one, that this set of requierments is for use with apples APNS, the WebPush(WP) section is lower down  and the pywebpush is declared there.
This line is a mistake in my eyes.